ios/RNCallKeep/RNCallKeep.m

```diff
@@ -133,8 +136,22 @@ + (void)initCallKitProvider {
     }
 }
 
++ (void)setup:(NSDictionary *)options {
+    RNCallKeep *callKeep = [RNCallKeep allocWithZone: nil];
+    [callKeep setup:options];
+    isSetupNatively = YES;
+}
+
 RCT_EXPORT_METHOD(setup:(NSDictionary *)options)
 {
+    if (isSetupNatively) {
+#ifdef DEBUG
+        NSLog(@"[RNCallKeep][setup] already setup");
+        RCTLog(@"[RNCallKeep][setup] already setup in native code");
+#endif
+        return;
```

Comment:
If we've already initialized from `AppDelegate.m`, we ignore further calls to setup from React Native.